yogstation/code/game/objects/items/fishing/bait.dm

```diff
@@ -30,20 +30,46 @@
 	if(prob(1))
 		desc = "Don't you dare say it"
 
+/*/obj/item/reagent_containers/food/snacks/bait/empowered //needed for the future, commented out for now
```

Comment:
what this for 👁

Reply:
was added before I found out about your fishing 2 :)